evaluation/scripts/wmt.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+
+import torch
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from ..datasets.wmt import WMTEnglishDataset
+
+
+def main(args):
+    """adapted from https://huggingface.co/transformers/perplexity.html"""
+    stride = args.stride
+    device = torch.device(args.device)
```

Comment:
We currently use this in simple benchmark script. 

```suggestion
    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```